proposals/PROPOSALS.md

```diff
@@ -1,10 +1,29 @@
-An HFTP (*Haskell Foundation Technical Proposal*) is a process for submitting a proposal for a technical initiative to that will be executed under the guidance of the Haskell Foundation (HF). This document describes how the community may propose, discuss, and implement technical initiatives to be carried out through the Haskell Foundation that will affect the Haskell ecosystem. Each HFTP goes through an iterative process in which it is discussed by the HF Technical Track (HFTT), the broader community, and any and all interested stakeholders. If a consensus is reached, the initiative is accepted and may then be executed by the Foundation (resources permitting). Only upon reaching a consensus are initiatives accepted and executed. Upon acceptance, an HFTP and its associated discussion are merged into this repository, and both serve as a historical document that details a specification and rationale for work detailed within the process.
+An HFTP (*Haskell Foundation Technical Proposal*) is a proposal for a technical initiative that will be executed under the guidance of the Haskell Foundation (HF). Each HFTP goes through an iterative process in which it is discussed by the HF Technical Working Group (TWG), the broader community, and any and all interested stakeholders. As part of this process, resources are identified that could ease or make possible the implementation of the proposal.
 
 The aim of the HFTP process is to apply the openness and collaboration that have shaped Haskell's documentation and implementation to a process of evolving the HF and the broader Haskell ecosystem. This document captures our guidelines, commitments and expectations regarding this process.
 
+## Acceptance
+
+Upon acceptance, an HFTP and its associated discussion are merged into this repository, and both serve as a historical document that details a specification and rationale for work detailed within the process. Acceptance of a proposal does not constitute a firm commitment by the Haskell Foundation to execute on the proposal, or to provide the necessary or helpful resources that were identified in the proposal. Fundamentally, acceptance indicates that the TWG believes that the Haskell ecosystem would be better off if the proposal were implemented, and that it is sufficiently well-considered.
```

Comment:
I suggest re-ordering.

Upon acceptance, an HFTP and its associated discussion are merged into this repository.  The accepted proposal serves both as a permanent document that details a specification, and rationale for work detailed within the process.

Fundamentally, acceptance indicates that the TWG believes that the Haskell ecosystem would be better off if the proposal were implemented, and that it is sufficiently well-considered.

However, acceptance of a proposal does not constitute a firm commitment by the Haskell Foundation to execute on the proposal, or to provide the necessary or helpful resources that were identified in the proposal.

-----------

Actually I'm not sure what "The accepted proposal serves both as a permanent document that details a specification, and rationale for work detailed within the process." means precisely.   Shall we just omit it entirely?

Reply:
That sounds good to me. I want to be explicit rather than implicit, but this text survived my edits without me really getting what it meant either. So I think it should go, or be revised if I've missed something important.